Documentation/ApiOverview/ContentElements/CustomBackendPreview.rst

```diff
@@ -92,3 +92,9 @@ approaches:
     when your extension is the one that creates the table, the latter is used
     when you need to override TCA properties of tables added by the Core or
     other extensions.
+
+..  note::
+    The content elements :php:`text`, :php:`textpic`, :php:`textmedia` and 
```

Comment:
What about bullets, etc?

Reply:
They do not have a special `PreviewRenderer` O.o 
You can check this in the Configuration module → $GLOBALS['TCA']  → search for "previewRenderer"